src/extension.ts

```diff
@@ -48,7 +48,7 @@ export async function activate(context: vscode.ExtensionContext) {
   try {
     playwrightTest = await PlaywrightTest.create(workspaceFolder.uri.fsPath, configuration.get('playwright.cliPath')!, debugModeHandler);
   } catch (error) {
-    vscode.window.showWarningMessage(error.toString());
+    console.warn(error.toString());
```

Comment:
We have a logger, I think its logger.debug then so it gets showed inside the output panel.

Reply:
Updated